File: fisheye_android/app/src/main/java/com/example/fisheye/MainActivity.java
package com.example.fisheye;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // FisheyeViewをメイン画面に配置
        FisheyeView fisheyeView = new FisheyeView(this);
        setContentView(fisheyeView);
        //setContentView(R.layout.activity_main);
    }
}

File: fisheye_kotlin/app/src/main/java/com/example/fisheye/MainActivity.kt
package com.example.fisheye

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // FisheyeViewをメイン画面に配置
        val fisheyeView = FisheyeView(this)
        setContentView(fisheyeView)
        //setContentView(R.layout.activity_main);
    }
}